src/test/java/tool/designpatterns/verifiers/multiclassverifiers/AdapterVerifierTest.java

```diff
@@ -0,0 +1,100 @@
+package tool.designpatterns.verifiers.multiclassverifiers;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.gradle.internal.impldep.org.junit.Assert.assertFalse;
+import static org.gradle.internal.impldep.org.junit.Assert.assertTrue;
+
+import com.github.javaparser.ast.NodeList;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.type.ClassOrInterfaceType;
+
+import org.junit.jupiter.api.Test;
+import tool.designpatterns.Pattern;
+import utilities.TestHelper;
+
+public class AdapterVerifierTest {
```

Comment:
From a glans it looks like you might be able to extract some of the code from multiple test methods to a separate, reusable method for more easily read code.

Reply:
It is already extracted and avaliable in this class https://github.com/Bachelor-Thesis-Project-Gruppun/StaticAnalyser/blob/develop/src/test/java/utilities/TestHelper.java

Reply:
Fixed